_layouts/about.html

```diff
@@ -26,6 +26,7 @@ <h1 class="post-title">
               {% include figure.html
               path=profile_image_path
               class=profile_image_class
+              sizes = "(min-width: 800px) 231, (min-width: 576px) 30vw, 95vw"
```

Comment:
should this be `230px`?

Reply:
yes, it should be 231px thanks. But I'm remembering that the 800px max width is easily variable in the config file... so I should probably use liquid to base it on that instead, I'll go do that.